content/applications/productivity/iot/config/updating_iot.rst

```diff
@@ -0,0 +1,150 @@
+(Internet of Things)` box , the term 'updating' can mean several different things. The actual
+drivers can be updated, the core code on the :abbr:`IoT (Internet of Things)` box can be updated
+and, lastly a new image can be flashed (physical :abbr:`IoT (Internet of Things)` box). This
+document will explore the various ways to update the :abbr:`IoT (Internet of Things)` box to ensure
+smooth operation of :abbr:`IoT (Internet of Things)` box processes and devices.
+
+Handlers (drivers) update
+=========================
+
+There may be some instances where the drivers or interfaces need to be updated for individual
+devices (e.g. scale, measurement tools, etc.). The IoT handlers (drivers and interfaces) code can be
+modified by syncing them with the configured server handler's code. This can be helpful in instances
+where :abbr:`IoT (Internet of Things)` devices (e.g. scales, measurement tools, etc.) are not
+working properly with the :abbr:`IoT (Internet of Things)` box.
+
+For both the Windows :abbr:`IoT (Internet of Things)` (Odoo 16 and higher) and physical :abbr:`IoT
+(Internet of Things)` box, this process can be performed manually from the :abbr:`IoT (Internet of
+Things)` box homepage. Go to the :abbr:`IoT (Internet of Things)` box homepage by navigating to
+:menuselection:`IoT app --> IoT Boxes` and clicking on the :guilabel:`IP address` of the :abbr:`IoT
+(Internet of Things)` box.
+
+Next, click :guilabel:`Handlers list` and then select :guilabel:`Load Handlers` at the bottom of the
+page.
+
+.. image:: updating_iot/load-handlers.png
+   :align: center
+   :alt: Handlers list on an IoT box with the load handlers button highlighted.
+
+.. important::
+   Handlers code is fetched from the configured server and it needs to be up-to-date to have the
+   latest fixes and patches. See :ref:`upgrade/homepage_upgrade`.
+
+.. note::
+   The handlers update is also performed automatically each time the :abbr:`IoT (Internet of
+   Things)` box is restarted. The only exception to this process is if the :guilabel:`Automatic
+   drivers update` is unchecked in the form view of the :abbr:`IoT (Internet of Things)` box on the
+   Odoo server. This setting can be reached by going to :menuselection:`IoT App --> Select the IoT
+   box --> Automatic drivers update`.
+
+.. _upgrade/homepage_upgrade:
+
+Update from the IoT box home page
+=================================
+
+.. important::
+   This update does not apply to the Windows :abbr:`IoT (Internet of Things)` box (Odoo 16 and
+   higher). To update the Windows :abbr:`IoT (Internet of Things)`, first, uninstall the previous
+   version of the Odoo Windows program and then reinstall it, with the most up-to-date installation
+   package. To begin the installation, navigate to the Odoo 16 or higher installation package for
+   Enterprise or Community - Windows edition at `Odoo’s download page <https://odoo.com/download>`_.
```

Comment:
the idea is interesting, but it forces you to repeat yourself on this section and the following and might change with time.
Maybe it would be better to have a dedicated section Update/"Reflash" Windows Iot with more detailed step on how to uninstall and re-install it ?

Reply:
Noted, this will be a seperate PR for 16.0 where windows IOT is introduced.